core/src/main/java/discord4j/core/object/entity/Member.java

```diff
@@ -253,4 +255,42 @@ public String getNicknameMention() {
         return getServiceMediator().getRestClient().getGuildService()
                 .removeGuildMemberRole(guildId, getId().asLong(), roleId.asLong());
     }
+
+    /**
+     * Requests to edit this member.
+     *
+     * @param spec A {@link Consumer} that provides a "blank" {@link GuildMemberEditSpec} to be operated on. If some
+     * properties need to be retrieved via blocking operations (such as retrieval from a database), then it is
+     * recommended to build the spec externally and call {@link #edit(GuildMemberEditSpec)}.
+     * @return A {@link Mono} where, upon successful completion, emits nothing; indicating the member has been edited.
+     * If an error is received, it is emitted through the {@code Mono}.
+     */
+    public Mono<Void> edit(final Consumer<GuildMemberEditSpec> spec) {
+        final GuildMemberEditSpec mutatedSpec = new GuildMemberEditSpec();
+        spec.accept(mutatedSpec);
+        return edit(mutatedSpec);
+    }
+
+    /**
+     * Requests to edit this member.
+     *
+     * @param spec A configured {@link GuildMemberEditSpec} to perform the request on.
+     * @return A {@link Mono} where, upon successful completion, emits nothing; indicating the member has been edited.
+     * If an error is received, it is emitted through the {@code Mono}.
+     */
+    public Mono<Void> edit(final GuildMemberEditSpec spec) {
+        return getServiceMediator().getRestClient().getGuildService()
+                .modifyGuildMember(getGuildId().asLong(), getId().asLong(), spec.asRequest());
+    }
+
+    /**
+     * Requests to change the nickname of this member.
+     *
+     * @param nickname The nickname to change the member to. May be null.
+     * @return A {@link Mono} where, upon successful completion, emits nothing; indicating the nickname has been
+     * changed. If an error is received, it is emitted through the {@code Mono}.
+     */
```

Comment:
Add an @see tag forwarding to the technically preferred signatures with GuildMemberEditSpec